From aaae59ca83940ed61563a8d9f7ba8674f72d885d Mon Sep 17 00:00:00 2001
From: Jeremy Fitzhardinge <jeremy.fitzhardinge@citrix.com>
Date: Thu, 2 Sep 2010 21:25:10 -0700
Subject: [PATCH 06/26] xen/evtchn: rename enable/disable_dynirq ->
 unmask/mask_irq

The functions are generic, so they may as well have generic names.

Signed-off-by: Jeremy Fitzhardinge <jeremy.fitzhardinge@citrix.com>
(cherry picked from commit d0936845a856816af2af48ddf019366be68e96ba)

Signed-off-by: Ian Campbell <ijc@hellion.org.uk>
---
 drivers/xen/events.c |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/xen/events.c b/drivers/xen/events.c
index d70e629..c0c69ff 100644
--- a/drivers/xen/events.c
+++ b/drivers/xen/events.c
@@ -1178,7 +1178,7 @@ int resend_irq_on_evtchn(unsigned int irq)
 	return 1;
 }
 
-static void enable_dynirq(unsigned int irq)
+static void unmask_irq(unsigned int irq)
 {
 	int evtchn = evtchn_from_irq(irq);
 
@@ -1186,7 +1186,7 @@ static void enable_dynirq(unsigned int irq)
 		unmask_evtchn(evtchn);
 }
 
-static void disable_dynirq(unsigned int irq)
+static void mask_irq(unsigned int irq)
 {
 	int evtchn = evtchn_from_irq(irq);
 
@@ -1366,9 +1366,9 @@ void xen_irq_resume(void)
 static struct irq_chip xen_dynamic_chip __read_mostly = {
 	.name		= "xen-dyn",
 
-	.disable	= disable_dynirq,
-	.mask		= disable_dynirq,
-	.unmask		= enable_dynirq,
+	.disable	= mask_irq,
+	.mask		= mask_irq,
+	.unmask		= unmask_irq,
 
 	.eoi		= ack_dynirq,
 	.set_affinity	= set_affinity_irq,
@@ -1378,9 +1378,9 @@ static struct irq_chip xen_dynamic_chip __read_mostly = {
 static struct irq_chip xen_percpu_chip __read_mostly = {
 	.name		= "xen-percpu",
 
-	.disable	= disable_dynirq,
-	.mask		= disable_dynirq,
-	.unmask		= enable_dynirq,
+	.disable	= mask_irq,
+	.mask		= mask_irq,
+	.unmask		= unmask_irq,
 
 	.ack		= ack_dynirq,
 };
-- 
1.7.5.4

